docs/core/development/policy/coding-standard.rst

```diff
@@ -625,6 +625,126 @@ Any attribute, method argument, or method name that corresponds *directly* to a
 All SQL keywords should be in upper case.
 
 
+Emitting logs
+-------------
+
+All new code emitting log messages should use the new logging system.
+
+This section is dedicated to using the new logging system in Twisted core code.
+For general information about using the logging system please check the dedicated documentation about :doc:`how to log with twisted.logger </core/howto/logger>` and the :api:`twisted.logger.Logger <Logger>` API docs.
+
+General usage rules:
+
+* Don't use explicit namespaces.
```

Comment:
This should be explained positively; "allow the logger to fill in the namespace".  But also: why?  We want to allow the logger to fill in the namespace *so that* it matches the implementation module.  Except, sometimes we don't; for a module like `twisted.logger` or `twisted._threads` we might actually prefer to promote to a higher namespace where the class "appears" to be defined. So sometimes it's quite reasonable to use an explicit namespace.  Given that fact, it seems a little odd to include this, and especially odd to include it as the very first rule.